pinc/LPage.inc

```diff
@@ -283,9 +283,10 @@ class RoundInfo
  */
 class LPage
 {
-    public function __construct($projectid, $imagefile, $page_state, $reverting_to_orig)
+    public function __construct($project, $imagefile, $page_state, $reverting_to_orig)
     {
-        $this->projectid = $projectid;
+        $this->project = $project;
+        $this->projectid = $project->projectid;
```

Comment:
would it make sense to just remove this property so we don't have to worry about keeping the `project` and `projectid` in sync?

Reply:
I suppose it would.

Reply:
The reasons for not doing that are that it could be slower (insignificant) and needs more changes, but there seems little risk of breaking anything without noticing so I'll do that.

Reply:
PPage uses LPage->projectid. I don't want to change PPage now so don't do this now. If in the future we use proofreading API and don't use PPage we can do it then.